GameStore.Infrastructure/Persistence/Seed/DatabaseSeeder.cs

```diff
@@ -0,0 +1,156 @@
+﻿using GameStore.Application.Persistence;
+using GameStore.Domain.Entities;
+
+namespace GameStore.Infrastructure.Persistence.Seed;
+
+public static class DatabaseSeeder
+{
+    public static async Task SeedDatabase(IUnitOfWork unitOfWork)
+    {
+        if (!(await unitOfWork.GenreRepository.GetAsync()).Any())
```

Comment:
https://learn.microsoft.com/ru-ru/ef/core/modeling/data-seeding

Reply:
You suggest using ModelBuilder.Entity<T>.HasData()?

Reply:
Yes, preferred, but If you want you can keep it as is